crates/scarb-api/src/lib.rs

```diff
@@ -126,29 +129,37 @@ pub fn get_contracts_map(
         profile.unwrap_or(metadata.current_profile.as_str()),
     )?;
 
-    let map = match maybe_contracts_path {
-        Some(contracts_path) => load_contract_artifacts(&contracts_path)?,
-        None => HashMap::default(),
+    let maps = match maybe_contracts_path {
+        Some(contracts_path) => load_contracts_artifacts_and_sierra_paths(&contracts_path)?,
+        None => (HashMap::default(), HashMap::default()),
     };
-    Ok(map)
+
+    Ok(maps)
 }
 
-fn load_contract_artifacts(
+fn load_contracts_artifacts_and_sierra_paths(
     contracts_path: &Utf8PathBuf,
-) -> Result<HashMap<String, StarknetContractArtifacts>> {
+) -> Result<(
+    HashMap<String, StarknetContractArtifacts>,
+    HashMap<String, Utf8PathBuf>,
```

Comment:
Why not keep it in a one map? Doesn't make sense to split it, and it makes for more complicated code

Reply:
Will change the interface